themes/grass/layouts/learn/list.html

```diff
@@ -78,12 +78,12 @@ <h2 class="section-title">Get to know the GRASS GIS ecosystem</h2>
  <a href="#" class="px-4 py-4 d-block bg-white htl"><h3 class="mt-2 mb-4"> Coming soon</h3><br><br><br></a>
   </div>
 
-  <div class="col-lg-4 col-sm-6 mb-4">
- <a href="#" class="px-4 py-4 d-block bg-white htl"><h3 class="mt-2 mb-4"> Coming soon</h3><br><br><br></a>
+  <div class="col-lg-4 col-sm-6 mb-4 text-center">
+ <a href="./learn/tryonline" class="px-4 py-4 d-block bg-white htl"><h3 class="mt-2 mb-4"> Try online </h3><img src="../images/www.svg" width="50%" alt=""  align="middle"/></a>
```

Comment:
this seems wrong, try with
```suggestion
 <a href="./tryonline" class="px-4 py-4 d-block bg-white htl"><h3 class="mt-2 mb-4"> Try online </h3><img src="../images/www.svg" width="50%" alt=""  align="middle"/></a>
```